public/app/plugins/datasource/loki/components/LokiQueryField.tsx

```diff
@@ -212,24 +211,35 @@ export class LokiQueryField extends React.PureComponent<LokiQueryFieldProps, Lok
                   <Icon name={labelBrowserVisible ? 'angle-down' : 'angle-right'} />
                 </button>
                 <div className="gf-form gf-form--grow flex-shrink-1 min-width-15">
-                  <QueryField
-                    additionalPlugins={this.plugins}
-                    cleanText={cleanText}
-                    query={query.expr}
-                    onTypeahead={this.onTypeahead}
-                    onWillApplySuggestion={willApplySuggestion}
-                    onChange={this.onChangeQuery}
-                    onBlur={this.props.onBlur}
-                    onRunQuery={this.props.onRunQuery}
-                    placeholder={placeholder}
-                    portalOrigin="loki"
-                  />
+                  {config.featureToggles.lokiMonacoEditor ? (
+                    <MonacoQueryFieldWrapper
+                      runQueryOnBlur={this.props.app !== CoreApp.Explore}
```

Comment:
Nit/personal preference I'd destructure props and just do  `runQueryOnBlur={app !== CoreApp.Explore}` everywhere.

Reply:
Sure.